### DISCUSSION TITLE
Is there anyone else who needs phase-nr to be configured - eg apply AC1 as phase-T

### DISCUSSION THREAD

**Discussion by silliewimons** [SBFspot/SBFspot#526] [General]
request for comments:  
environment:   while monitoring a site with two single-phase inverters, 
each of the two inverters is connected to another phase (to spread the production)   inverter1 => R  inverter2 => T
i found it a draw-back that i could not instruct sbfspot to report power from second inverter to be 'noted' as Pac3 

when this should be available then one could merge spot-outputs for the 2 inverters into one report,

note: a related point is "identification of the phases,  most documents refer to the phases as:   R, S, T  or  L1, L2, L3
in belgium/netherland/luxemb. (and ??other countries)   in technical instructions there is since 1987 a preference for L1, L2, L3 
the below text shows the common accepted conventions  (older <-> to-day's convention)        
R= Reference     S= Secondary    T= Tertiary      new markings are   L1= R                 L2 = S                L3 = T
in some countries another (older) way to identify was RYB (red / yellow / blue) according an (old) colour schema for the wires.
since 2001 Europe applies L1=Brown  L2=Black  L3=Grey  N=Blue  as colour shema for the wiring

IF installers of inverters / battery systems do correctly apply the wiring, then with 3phase inverters there shouldn't be a problem.
kind regards   simons willy

**Comment by sbf-** [comment]
> when this should be available then one could merge spot-outputs for the 2 inverters into one report

This is all possible with SQL

**Comment by deagel** [comment]
Hi, 
you ask for view information:

> dear deagel, in your output log i could read that in your system
> 
> the AC-output from your mono-phase inverters was assigned to a dedicated phase
> sbfspot assigns ACoutput standard to phase1
> 
> can you provide more information on the way that you did realise this,
> see part: discussions item
> ...
> kind regards silliew

after more or less of trouble with LG to get a new Storage all my inverter are up again.

i have tree inverter connected to different ac phase:
Device Type:      SB 2500TLST-21
Device Type:      Sunny Boy 2.5
Device Type:      Sunny Boy Storage 3.7

SUSyID: 235 - SN: xxxxxxxxxxxxx
DC Spot Data:
        MPPT 1 Pdc:   1.669kW - Udc: 287.24V - Idc:  5.813A
        MPPT 2 Pdc:   0.000kW - Udc:   0.00V - Idc:  0.000A
        Calculated Total Pdc:   1.669kW
AC Spot Data:
        Phase 1 Pac :   1.617kW - Uac: 229.10V - Iac:  7.042A
        Phase 2 Pac :   0.000kW - Uac:   0.00V - Iac:  0.000A
        Phase 3 Pac :   0.000kW - Uac:   0.00V - Iac:  0.000A
        Total Pac   :   1.613kW - Calculated Pac:   1.617kW
        Efficiency  :   96.88%
SUSyID: 303 - SN: yyyyyyyyyyy
DC Spot Data:
        MPPT 1 Pdc:   1.591kW - Udc: 208.23V - Idc:  7.641A
        MPPT 2 Pdc:   0.000kW - Udc:   0.00V - Idc:  0.000A
        Calculated Total Pdc:   1.591kW
AC Spot Data:
        Phase 1 Pac :   0.000kW - Uac:   0.00V - Iac:  0.000A
        Phase 2 Pac :   1.527kW - Uac: 232.95V - Iac:  6.558A
        Phase 3 Pac :   0.000kW - Uac:   0.00V - Iac:  0.000A
        Total Pac   :   1.527kW - Calculated Pac:   1.527kW
        Efficiency  :   95.98%
SUSyID: 361 - SN: zzzzzzzzz
DC Spot Data:
        MPPT 1 Pdc:   0.000kW - Udc:   0.00V - Idc:  0.000A
        MPPT 2 Pdc:   0.000kW - Udc:   0.00V - Idc:  0.000A
        Calculated Total Pdc:   0.000kW
AC Spot Data:
        Phase 1 Pac :   0.000kW - Uac:   0.00V - Iac:  0.000A
        Phase 2 Pac :   0.000kW - Uac:   0.00V - Iac:  0.000A
        Phase 3 Pac :  -2.378kW - Uac: 223.87V - Iac: 10.638A
        Total Pac   :  -2.379kW - Calculated Pac:  -2.378kW
        Efficiency  :    0.00%


Iam not sure if I got you right but selecting the phase (Germany L1,L2,L3) was done while the installation process as you know. I'm only grep the data with sbf from my inverter and push it to pvoutput. 

what I miss is a possibility to grep data from sma home manager 2 to have a combined view like in sunny portal. For bigger reports I use sunny portal and or my Tigo data.

**Comment by silliewimons** [comment]
dear deagel    -  thank you for your reply     :-)      yet another way to look to A problem

note-1: in my system (sun-explorer & sb3300) i do not have that 'config-option' -> select 'ACphase' to identify that relation
             BUT from a system  (or SMA's) viewpoint this is a correct option 'having 3phases  & mono-phase inverters and HomeManager
yet ?? another point on ?list? of items to investigate
   
note-2: each member of sbfspot community can only refer to its own-systems for 'experience'
         i'm gracefull on receiving your contribution  to  this item,

about your remarks:    
 ?? getting/grepping data from  "sma home manager"   i include the energy-meter   
 ?? view like in sunny-portal   
in my humble opinion -> open source projects like sbfspot  contribute in PUSHing sma-developpers  to improve  the systems

sincerely yours   silliew

**Comment by silliewimons** [comment]
dear contributors,   as indicated by deagel,               "additional-information"    
- SBFspot   uses the information  received  from  "inverter" in function of phase assigned  to  ACoutput

this option/function might be "inverter" dependent,
this has an impact when using/interpreting information from 'Energy-meter' | 'Home-Manager' | . . . .   
kr silliew
